make/compiler/Makefile.cray-prgenv

```diff
@@ -60,6 +60,10 @@ else ifeq ($(CHPL_MAKE_COMM), ofi)
 # dynamic libraries will be required at execution time, so default to
 # dynamic linking to avoid these warnings.
 RUNTIME_LFLAGS += -dynamic
+else ifeq ($(CHPL_MAKE_COMM), ugni)
+ifneq ($(UGNI_PERFSTATS),)
+RUNTIME_LFLAGS += -dynamic
```

Comment:
Might add a one-liner comment here mentioning the desirability of enabling vDSO.